### DISCUSSION TITLE
What is the correct way to check if the value is null or not?

### DISCUSSION THREAD

**Discussion by thinhbuzz** [vfsfitvnm/frida-il2cpp-bridge#162] [Q&A]
Now I am using below code to check data is null or not, but sometimes it is not working, isNull is false but in console print data is null. I know, it's hard to test without proof, but now I only have this much information, hope you can take a look.

```typescript
console.log(data);
const isNull = !data || data.isNull() || data.toString() == 'null';
console.log(isNull);
```

**Comment by vfsfitvnm** [comment]
`isNull` should be sufficient: https://github.com/vfsfitvnm/frida-il2cpp-bridge/blob/a05a34851920e9826a22120c1babd1af89a5b6fd/src/utils/native-struct.ts#L18
Also, `toString` returns `"null"` when the returned string is `null`: https://github.com/vfsfitvnm/frida-il2cpp-bridge/blob/a05a34851920e9826a22120c1babd1af89a5b6fd/src/il2cpp/structs/object.ts#L80
